src/electionguard_gui/services/key_ceremony_service.py

```diff
@@ -68,7 +71,7 @@ def watch_key_ceremonies(
                 # the tailable cursor times out after a few seconds and fires a StopIteration exception,
                 # so we need to catch it and restart watching. The sleep is required by eel to allow
                 # it to respond to events such as the very important stop_watching event.
-                eel.sleep(0.2)
+                eel.sleep(0.8)
```

Comment:
spending more time in the eel sleep makes the UI substantially more responsive, and the .2s long polling is still enough to give super-fast feedback cycles during the multiple back-and-forth's of the key ceremony.